src/reanimated2/NativeReanimated/NativeReanimated.ts

```diff
@@ -90,9 +90,8 @@ export class NativeReanimated {
     }
     global._REANIMATED_VERSION_JS = jsVersion;
     if (global.__reanimatedModuleProxy === undefined) {
-      const { ReanimatedModule } = NativeModules;
       const valueUnpackerCode = getValueUnpackerCode();
-      ReanimatedModule?.installTurboModule(valueUnpackerCode);
+      NativeReanimatedModule?.installTurboModule(valueUnpackerCode);
```

Comment:
This is probably out of the scope of this PR but it's quite confusing that we call `installTurboModule` method on an existing turbo module. We should probably rename `installTurboModule` to `installJSIBindings`. Alternatively, we could even use `ReanimatedModule` directly instead of `this.InnerNativeModule` (but let's do it in a separate PR).

Reply:
>  We should probably rename installTurboModule to installJSIBindings.

I can do it for this PR and the rest of the work in the next one. Let's merge it ASAP.